staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/validation/validation.go

```diff
@@ -48,7 +49,7 @@ var (
 )
 
 // ValidateCustomResourceDefinition statically validates
-func ValidateCustomResourceDefinition(obj *apiextensions.CustomResourceDefinition) field.ErrorList {
+func ValidateCustomResourceDefinition(obj *apiextensions.CustomResourceDefinition, ctx context.Context) field.ErrorList {
```

Comment:
Why are we passing in context for CRD validation? I was expecting we'd just need it for CR validation (but I'm assuming I'm missing something here).

Reply:
Because we still need to run celValidator.Validate for default while CRD validation process.

Reply:
godoc added for all affected func